### DISCUSSION TITLE
I'm getting rotating spinners on numerous places of all github webpages. How to fix it?

### DISCUSSION THREAD

**Discussion by oldunixguy** [community/community#40221] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I'm running firefox quantum 66.0.3 and for a long time I have had NO problems with github use. Recently many aspects of github are NOT working. for example the "Selection" button above this dialog used to select a topic area will only drop down a list but I cannot select and item. I cannot add a screenshot image to discussion topics (to show these issues). When I select the "edit" button on a discussion item I wrote the button changes color but nothing else happens. When I select the "Category" change button on a discussion I wrote a small box appears but it contains a rotating spinner. When I go to a particular github page (https://github.com/inkstitch/inkstitch/releases/tag/v1.29.1) and go to the Assets section all that shows is a rotating spinner. this stuff used to work! Any idea on how to cure this as it has completely broken my ability to use github as well as not able to download any github project software.
thanks
oldunixguy

**Comment by airtower-luna** [comment]
I assume it's simply incompatibility with a *vastly* outdated browser. The current Firefox version is 107, or if you prefer the extended support release (ESR, with major changes less often) it's 102.5. You really should upgrade your Firefox, also because I'm pretty sure there were a lot of fixed security issues since.

**Comment by oldunixguy** [comment]
thanks. would love to but I'm stuck here because I'm stuck on Ubuntu 14.04
lts for a long list of dependencies.
isnt github web w3c.org standards compliant? if it is the older browsers
should work fine.
But this worked as recently as a month or so ago....
oh well. i'm screwed.
thanks
oldunixguy

On Sun, Nov 27, 2022 at 1:29 PM Airtower ***@***.***> wrote:

> I assume it's simply incompatibility with a *vastly* outdated browser.
> The current Firefox version is 107, or if you prefer the extended support
> release (ESR, with major changes less often) it's 102.5. You really should
> upgrade your Firefox, also because I'm pretty sure there were a lot of
> fixed security issues since.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/40221#discussioncomment-4248262>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADBQBDZUYYVELBQXTWGKODTWKPADTANCNFSM6AAAAAASMVYIGY>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Richard A. Painter, P.E. Retired

**Comment by airtower-luna** [comment]
> thanks. would love to but I'm stuck here because I'm stuck on Ubuntu 14.04
> lts for a long list of dependencies.

That sounds like you're in an IT security horror story. :fearful: Not only your browser is long out of security support, but the whole system. If you absolutely can't update, you need to separate the system from the internet (maybe move it into a VM), and use a separate machine for anything that doesn't strictly require the old software stack. I wish I could say more, but that'd require knowing more.

If it's a work thing and someone simply refuses to approve the effort of making whatever it is work on a current system, I'd recommend finding a new job. If they refuse that, I suspect they won't take responsibility *when* things finally break.

**Comment by oldunixguy** [comment]
I understand. I have a newer version of Chromium Version 65.0.3325.181 and that acts the SAME way. Hmm so its not just firefox.
thanks
oldunixguy
